### PR TITLE
procedures: starting a new workspace

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change
Adding the first batch of new content to the new docs.

## What issues does this pull request fix or reference
RHDEVDOCS-3505 > RHDEVDOCS-3789, RHDEVDOCS-3790

## Specify the version of the product this pull request applies to
7.42+

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/user-guide/pages/parameters-for-the-urls-for-starting-a-new-workspace.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+:keywords: url, urls, parameter, parameters, starting, start, workspace, workspaces
+:navtitle: Parameters for the URLs for starting a new workspace
+// :page-aliases:
+
+[id="parameters-for-the-urls-for-starting-a-new-workspace_{context}"]
+= Parameters for the URLs for starting a new workspace
+
+* xref:url-parameter-for-the-workspace-ide.adoc[]
+* xref:url-parameter-for-starting-duplicate-workspaces.adoc[]
+* xref:url-parameter-for-devfile-file-name.adoc[]
```
Comment (themr0c) [inline modules/user-guide/pages/parameters-for-the-urls-for-starting-a-new-workspace.adoc]:
```suggestion
* xref:url-parameter-for-the-devfile-file-name.adoc[]
```

File: modules/user-guide/pages/parameters-for-the-urls-for-starting-a-new-workspace.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* xref:url-parameter-for-devfile-file-path.adoc[]
```
Comment (themr0c) [inline modules/user-guide/pages/parameters-for-the-urls-for-starting-a-new-workspace.adoc]:
```suggestion
* xref:url-parameter-for-the-devfile-file-path.adoc[]
```

File: modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:description: Starting a new workspace with a clone of a Git repository
+:keywords: start-new-workspace, start-a-new-workspace, how-to-start-new-workspace, how-to-start-a-new-workspace, starting-a-new-workspace, clone-git-repository, clone-a-git-repository, how-to-start-workspace, how-to-start-a-workspace
+:navtitle: Starting a new workspace with a clone of a Git repository
+// :page-aliases:
+
+[id="starting-a-new-workspace-with-a-clone-of-a-git-repository_{context}"]
+= Starting a new workspace with a clone of a Git repository
+
+With {prod-short}, you can start a new workspace that contains a clone of a Git repository. So you can have your remote origin or fork cloned into a new starting workspace. Or you can have a common repository of a project cloned into a new starting workspace. With such Git providers as GitHub and GitLab, you can also specify the branch for the new starting workspace to checkout.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc:10]:
```suggestion
With {prod-short}, you can start a new workspace that contains a clone of a Git repository. You can have your remote origin or fork cloned into a new starting workspace. You can also have a common repository of a project cloned into a new starting workspace. With Git providers such as GitHub and GitLab, you can also specify the branch for the new starting workspace to checkout.
```

Comment (swatikarmarkar) [inline modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc:10]: The {prod-short} variable is not populated with the correct value.

File: modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+TIP: Advanced users can compose and use their own devfiles, as well as use a local devfile.
+//provide links on "compose", "devfiles", and "local devfile" to the corresponding sections in Advanced Use (User Guide). max-cx
+
+IMPORTANT: Although the devfile enables full functionality of a workspace, you can also start a new workspace by supplying the URL of a Git repository without a devfile. This results in a new workspace with a clone of the Git repository and with the defaults such as the Che-Theia IDE and the Universal Developer Image.
+//provide xrefs to a concept module about the Universal Developer Image and, if applicable, devfile-less defaults for new workspaces. max-cx
+
+NOTE: The *Create Workspace* page of your {prod-short} xref:url-of-your-che-dashboard.adoc[dashboard] has the *Git Repo URL ** field as the GUI alternative for entering the URL of a Git repository to start a new workspace.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc:18]:
```suggestion
NOTE: The *Create Workspace* page of your {prod-short} xref:url-of-your-che-dashboard.adoc[dashboard] has the **Git Repo URL** field as the GUI alternative for entering the URL of a Git repository to start a new workspace.
```

File: modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+.Prerequisites
+
+* Your organization has a running instance of {prod-short}.
+* You have been given the xref:url-of-your-organizations-che-instance.adoc[].
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc:23]:
```suggestion
* You are provided with the xref:url-of-your-organizations-che-instance.adoc[].
```

File: modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+* Your Git repository maintainer keeps the `devfile.yaml` or `.devfile.yaml` file in the root directory of the Git repository.
+* Optional: Private repositories that are hosted on GitHub, GitLab, or Bitbucket require (1) an authorized application link to be created on the repository-hosting website by the repository maintainer (which is you for your fork) and (2) a {platforms-name} secret to be applied by the administrator of the {prod-short} instance that belongs to your organization.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc:25]:
```suggestion
* Optional: Private repositories that are hosted on GitHub, GitLab, or Bitbucket require:
- An authorized application link to be created on the repository-hosting website by the repository maintainer (which is you for your fork) and
- A {platforms-name} secret to be applied by the administrator of the {prod-short} instance that belongs to your organization.

```

File: modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+//two inline links to be added into this par in the next, separate PR. max-cx
+
+.Procedure
+
+. Optional: visit the xref:url-of-your-che-dashboard.adoc[] to log in as a {prod-short} user to your organization's instance of {prod-short}.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc:30]:
```suggestion
. Optional: Visit the xref:url-of-your-che-dashboard.adoc[] to log in as a {prod-short} user to your organization's instance of {prod-short}.
```

File: modules/user-guide/pages/url-of-your-che-dashboard.adoc [hunk 8]
```diff
@@ -0,0 +1,21 @@
+:_content-type: CONCEPT
+:description: URL of your {prod-short} dashboard
+:keywords: dashboard, manage-workspaces, verify, workspace-status, workspace-state
+:navtitle: URL of your {prod-short} dashboard
+// :page-aliases:
+
+[id="url-of-your-{prod-id-short}-dashboard_{context}"]
+= URL of your {prod-short} dashboard
+
+The *Workspaces* page of your {prod-short} dashboard is where you manage your workspaces and verify their current states. The URL syntax for this page is like so:
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/url-of-your-che-dashboard.adoc:10]:
```suggestion
The *Workspaces* page of your {prod-short} dashboard is where you manage your workspaces and verify their current states. The URL syntax for this page is as follows:
```

File: modules/user-guide/pages/url-parameter-for-starting-duplicate-workspaces.adoc [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+Visiting a URL for starting a new workspace results in a workspace that is based on a combination of a devfile and a linked Git repository.
+
+In some situations, you may need to start one or more workspaces from the same URL for starting a new workspace, resulting in workspaces that are duplicates in terms of the same devfile and the same linked Git repository. The URL parameter for starting a duplicate workspace is `new`. The URL for starting a new workspace has the following syntax with the `new` parameter:
+
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?new
+----
+
+NOTE: Attempting to reuse the devfile and linked Git repository without the `new` URL parameter to start a new workspace in addition to a workspace that already exists results in an error message.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/url-parameter-for-starting-duplicate-workspaces.adoc:19]:
```suggestion
NOTE: Attempting to reuse the devfile and linked Git repository without the `new` URL parameter to start a new workspace in addition to a workspace that already exists will provide an error message.
```

File: modules/user-guide/pages/url-parameter-for-the-devfile-file-name.adoc [hunk 10]
```diff
@@ -0,0 +1,24 @@
+:_content-type: CONCEPT
+:description: URL parameter for the devfile file name
+:keywords: devfile, override-devfile-file-name, override-file-name, file-name, devfile-name, different-devfile-name
+:navtitle: URL parameter for the devfile file name
+// :page-aliases:
+
+[id="url-parameter-for-the-devfile-file-name_{context}"]
+= URL parameter for the devfile file name
+
+When you visit a URL for starting a new workspace, {prod-short} searches the linked Git repository for a devfile with the file name `.devfile.yaml` or `devfile.yaml`. So the devfile in the linked Git repository must follow this convention.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/url-parameter-for-the-devfile-file-name.adoc:10]:
```suggestion
When you visit a URL for starting a new workspace, {prod-short} searches the linked Git repository for a devfile with the file name `.devfile.yaml` or `devfile.yaml`. Therefore, the devfile in the linked Git repository must follow the convention.
```

File: modules/user-guide/pages/url-parameter-for-the-devfile-file-path.adoc [hunk 11]
```diff
@@ -0,0 +1,22 @@
+:_content-type: CONCEPT
+:description: URL parameter for the devfile file path
+:keywords: devfile, file-path, devfile-path, devfile-file-path, url-parameter, devfile-path-parameter, devfile-file-path-parameter
+:navtitle: URL parameter for the devfile file path
+// :page-aliases:
+
+[id="url-parameter-for-the-devfile-file-path_{context}"]
+= URL parameter for the devfile file path
+
+When you visit a URL for starting a new workspace, {prod-short} searches the root directory of the linked Git repository for a devfile (with the file name `.devfile.yaml` or `devfile.yaml`). So the file path of the devfile in the linked Git repository must follow this convention.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/url-parameter-for-the-devfile-file-path.adoc:10]:
```suggestion
When you visit a URL for starting a new workspace, {prod-short} searches the root directory of the linked Git repository for a devfile (with the file name `.devfile.yaml` or `devfile.yaml`). Therefore, the file path of the devfile in the linked Git repository must follow the convention.
```

File: modules/user-guide/pages/url-parameter-for-the-workspace-ide.adoc [hunk 12]
```diff
@@ -0,0 +1,44 @@
+:_content-type: CONCEPT
+:description: URL parameter for the workspace IDE
+:keywords: url-parameter-workspace-ide, workspace-ide, url-workspace-ide, url-parameter-workspace-editor, workspace-editor, url-workspace-editor
+:navtitle: URL parameter for the workspace IDE
+// :page-aliases:
+
+[id="url-parameter-for-the-workspace-ide_{context}"]
+= URL parameter for the workspace IDE
+
+Che-Theia is currently the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
```
Comment (swatikarmarkar) [inline modules/user-guide/pages/url-parameter-for-the-workspace-ide.adoc:10]:
```suggestion
Che-Theia is currently the default integrated development environment (IDE) loaded by a workspace if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
```

**Comment by max-cx** [comment]
Migrating the contents of this PR into a new PR for the master branch.

**Pull request closed without merging**